lib/Zonemaster/Backend/Config.pm

```diff
@@ -186,6 +186,50 @@ sub lock_on_queue {
     return $val;
 }
 
+=head2 timeout_before_retrying_test_in_minutes
+
+An optionnal parameter that allows to customize the period of time in minutes after which an unfinished test will be automatically terminated by the backend with a critical error.
```

Comment:
"optionnal" => "optional"

The name of the method and the description have different focuses. The PR was to resolve the issue when tests die before they are completed and hang uncompleted. That should also be reflected by the method name and the setting in the INI file.